.github/workflows/integration-tests.yml

```diff
@@ -238,22 +238,22 @@ jobs:
             github.com/smartcontractkit/chainlink-evm=${{ inputs.evm-ref }}
           gati-role-arn: ${{ secrets.AWS_OIDC_GLOBAL_READ_ONLY_TOKEN_ISSUER_ROLE_ARN }}
           gati-lambda-url: ${{ secrets.AWS_INFRA_RELENG_TOKEN_ISSUER_LAMBDA_URL }}
-
-  run-core-cre-e2e-tests-for-pr:
-    name: Run Core CRE E2E Tests For PR
-    permissions:
-      actions: read
-      checks: write
-      pull-requests: write
-      id-token: write
-      contents: read
-    needs: [build-chainlink, changes]
-    if: github.event_name == 'pull_request' && needs.changes.outputs.should-run-cre-tests == 'true'
-    uses: smartcontractkit/chainlink/.github/workflows/cre-system-tests.yaml@53e6348b792989514a6b7b53745bfcc389f706d9 # 27th June 2025
-    with:
-      chainlink_version: ${{ inputs.evm-ref || inputs.cl_ref || github.sha }}
-      chainlink_image_tag: ${{ inputs.evm-ref && format('{0}-plugins', inputs.evm-ref) || inputs.cl_ref && format('{0}-plugins', inputs.cl_ref) || format('{0}-plugins', github.sha) }}
-    secrets: inherit
+  # flaky test: https://chainlink-core.slack.com/archives/C07GQNPVBB5/p1758523002485979
+  # run-core-cre-e2e-tests-for-pr:
+  #   name: Run Core CRE E2E Tests For PR
+  #   permissions:
+  #     actions: read
+  #     checks: write
+  #     pull-requests: write
+  #     id-token: write
+  #     contents: read
+  #   needs: [build-chainlink, changes]
+  #   if: github.event_name == 'pull_request' && needs.changes.outputs.should-run-cre-tests == 'true'
+  #   uses: smartcontractkit/chainlink/.github/workflows/cre-system-tests.yaml@53e6348b792989514a6b7b53745bfcc389f706d9 # 27th June 2025
+  #   with:
+  #     chainlink_version: ${{ inputs.evm-ref || inputs.cl_ref || github.sha }}
+  #     chainlink_image_tag: ${{ inputs.evm-ref && format('{0}-plugins', inputs.evm-ref) || inputs.cl_ref && format('{0}-plugins', inputs.cl_ref) || format('{0}-plugins', github.sha) }}
+  #   secrets: inherit
```

Comment:
Is this the best way to skip it? What about the `if: false` trick? So it still shows up but as skipped?

Reply:
Ah the new approach uses `RUN_QUARANTINED_TESTS`: https://github.com/smartcontractkit/quarantine/blob/main/quarantine.go#L13